src/algorithms/panoc.jl

```diff
@@ -192,17 +192,19 @@ function Base.iterate(iter::PANOCIteration{R,Tx,Tf}, state::PANOCState) where {R
     copyto!(state.z_curr, state.z)
     state.f_Ax = state.f_Ax_d
 
+    # retrieve merit and set threshold
+    FBE_x = state.merit
     sigma = iter.beta * (0.5 / state.gamma) * (1 - iter.alpha)
     tol = 10 * eps(R) * (1 + abs(FBE_x))
     threshold = FBE_x - sigma * norm(state.res)^2 + tol
```

Comment:
Why do the assignment  `FBE_x = state.merit` here? It seems to be giving an inappropriate name to this quantity. Should you not directly use `state.merit` where needed in the lines that follow?

Reply:
You are right, I just wanted to minimize the changes relative to the monotone version. The assignment `FBE_x = state.merit` has been removed now with dbdd5c1, where possible.
In ZeroFPR and PANOCplus I left an assignment to `FBE_x` before the loop because the tests where failing otherwise; I did not understand why the variable `FBE_x` was not available at the final update of `state.merit`.